README.md

```diff
@@ -91,6 +91,19 @@ Default setup uses CPU (`--ngpu 0`).
 
 Note that if you want to use multi-gpu, the installation of nccl is required before setup.
 
+### Docker Container
+For working inside a docker container execute `run.sh` located inside the docker folder, it will builds a container and execute the given applicaiton. The execution line is detailed as following: 
```

Comment:
`For working inside a docker container execute `run.sh` located inside the docker folder, it will builds a container and execute the given applicaiton. `

To work inside a docker container, execute `run.sh` located inside the docker directory.
It will build a container and execute the main program specified by the following GPU, ASR example, and outside directory information, as follows: